models/common-label.json

```diff
@@ -5,7 +5,7 @@
   "type": "string",
   "minLength": 2,
   "maxLength": 255,
-  "pattern": "^[^;]{2,255}$",
+  "pattern": "^(?=.{2,255}$)(?=.*\\S.*\\S.*)[^;]*$",
```

Comment:
There are also other places where the (old) pattern is used. For example in query parameters (because we can't `$ref` those apparently), and in descriptions. We should probably update it everywhere consistently?

Reply:
Good point, I did a search and replace.